### DISCUSSION TITLE
SDXL second pass Hires resize

### DISCUSSION THREAD

**Discussion by toyssamurai** [vladmandic/sdnext#1846] [Q&A] [answered]
Does "hires resize" in second pass work with SDXL? Here's what I did:

1. Top drop down: Stable Diffusion checkpoint: 1.0 base
2. Top drop down: Stable Diffusion refiner: 1.0
3. Width and height set to 1024
4. Checked Second pass check box

The "Second pass" section showed up, but under the "Denoising strength" slider, I got:

**Hires resize: disabled**

No matter what I chose for the "Upscale by" slider, the image always came out as 1024 x 1024.

In addition, under the "Resize width to" slider, there's grayed out text saying:

**Refiner: disabled**

I did not see any error in the log.

**Comment by toyssamurai** [comment, marked as answer]
I guess I can answer my own question. Just found the following from the Wiki:

```
Explicit refiner as postprocessing is not yet implemented
Workflows such as hires fix are not yet implemented
```

I guess I was under the impression that hires and second pass refiner works because other people mentioned them in other threads.

**Reply by vladmandic** [reply]
hires **works** for original backend, but not diffuser backend
refiner **works** for diffuser backend, but not original backend
having full matrix is coming, but its a question of time.
